CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ project(v8pp
 	LANGUAGES CXX
 )
 
+set(V8PP_MAJOR 2)
+set(V8PP_MINOR 0)
+set(V8PP_REVISION 0)
+
```

Comment:
So it would no need to duplicate the project version components with already existing CMake variables.